easybuild/easyblocks/p/pytorch.py

```diff
@@ -152,9 +152,23 @@ def configure_step(self):
         if get_software_root('imkl'):
             options.append('BLAS=MKL')
             options.append('INTEL_MKL_DIR=$MKLROOT')
-        else:
-            # This is what PyTorch defaults to if no MKL is found. Make this explicit here
+        elif LooseVersion(self.version) >= LooseVersion('1.9.0') and get_software_root('BLIS'):
+            options.append('BLAS=BLIS')
+            options.append('BLIS_HOME=' + get_software_root('BLIS'))
+            options.append('USE_MKLDNN_CBLAS=ON')
```

Comment:
shouldn't this be added here?
```python
options.append('WITH_BLAS=blis')
```
see https://github.com/ashishfarmer/pytorch/commit/1dac97dfb4185b4a014bddd97ee58d0529a90900

Reply:
may not be strictly necessary but better to be explicit?

Reply:
Not really. First the WITH_BLAS is only used when BLAS=Eigen and second WITH_BLAS is not even passed to CMake at all which is why the hack at https://github.com/easybuilders/easybuild-easyblocks/pull/2448/files#diff-348e306def79b774c5b63a5086cc82b723524fcb348042b12559d6b82f9e99b9R167 is required.

So no I wouldn't add this until PyTorch made up their mind how they really want things to be. See https://github.com/pytorch/pytorch/issues/59218